step_shutdown.go

```diff
@@ -80,8 +76,9 @@ func (s *StepShutdown) Run(state multistep.StateBag) multistep.StepAction {
 	}
 
 	ui.Say("VM stopped")
+	state.Put(BuildSuccessFlag, true)
```

Comment:
Sorry, my fault: this should be removed, but I forgot to push the changed file. Pushed now.